[2025-03-04T14:44:57.743+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T14:44:57.782+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_command_dag.hello_world scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T14:44:57.804+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_command_dag.hello_world scheduled__2025-03-03T00:00:00+00:00 [queued]>
[2025-03-04T14:44:57.805+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-04T14:44:57.842+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): hello_world> on 2025-03-03 00:00:00+00:00
[2025-03-04T14:44:57.856+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bash_command_dag', 'hello_world', 'scheduled__2025-03-03T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpstt283ib']
[2025-03-04T14:44:57.858+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=184) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T14:44:57.860+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask hello_world
[2025-03-04T14:44:57.861+0000] {standard_task_runner.py:72} INFO - Started process 188 to run task
[2025-03-04T14:44:57.898+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-04T14:44:57.978+0000] {task_command.py:467} INFO - Running <TaskInstance: bash_command_dag.hello_world scheduled__2025-03-03T00:00:00+00:00 [running]> on host 60afe3d3ebe0
[2025-03-04T14:44:57.986+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-04T14:44:58.227+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_command_dag' AIRFLOW_CTX_TASK_ID='hello_world' AIRFLOW_CTX_EXECUTION_DATE='2025-03-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-03T00:00:00+00:00'
[2025-03-04T14:44:58.229+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T14:44:58.316+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-04T14:44:58.319+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello from Airflow!"']
[2025-03-04T14:44:58.333+0000] {subprocess.py:99} INFO - Output:
[2025-03-04T14:44:58.339+0000] {subprocess.py:106} INFO - Hello from Airflow!
[2025-03-04T14:44:58.340+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-04T14:44:58.382+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T14:44:58.384+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bash_command_dag, task_id=hello_world, run_id=scheduled__2025-03-03T00:00:00+00:00, execution_date=20250303T000000, start_date=20250304T144457, end_date=20250304T144458
[2025-03-04T14:44:58.445+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T14:44:58.492+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-04T14:44:58.495+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
